demo/demo2.js

```diff
@@ -0,0 +1,96 @@
+const { command, processText, Format, CommandResult } = require('../src');
+
+const readline = require('readline');
```

Comment:
You know, I've never actually done anything on the command-line with Node? Kinda crazy, considering I've been writing it for a year or 2 and that's where all of us started.

Reply:
:man_shrugging:, I'm not too surprised. Unless you're writing scaffolders there is little reason to, and even then a lot of them use selection lists instead of taking raw input.